app/src/main/java/com/garethevans/church/opensongtablet/StageMode.java

```diff
@@ -5245,6 +5251,10 @@ protected void onPreExecute() {
             try {
                 // We know how many columns we are using, so lets go for it.
                 column1_1 = processSong.preparePerformanceColumnView(StageMode.this);
+                // IV - If doing song block - needs own preference
+                if (!(preferences.getMyPreferenceString(StageMode.this, "stickyAutoDisplay", "F").equals("N"))) {
```

Comment:
No issues adding another preference.  Add it to the comment lines in the Preferences.java file (allows tracking of the variables currently in the app).